.github/workflows/e2e.yml

```diff
@@ -40,6 +40,10 @@ jobs:
         uses: actions/setup-go@v1
         with:
           go-version: 1.17
+      - name: Setup Node Version
+        uses: actions/setup-node@v2
+        with:
+          node-version-file: '.nvmrc'
```

Comment:
Given that the other actions use Nix, could e2e do the same?

Reply:
Yes, please!

I'd suggest all our automation (CI workflows) use Nix, and we additionally standardize on [asdf](https://asdf-vm.com) for local dev environments as it's powerful enough to support our polyglot use cases. It's also loved by many Chainlink engineers.

@tateexon why don't we turn this effort into:

1. Propagate Nix usage in CI workflows
2. Configure & document `asdf` local environments

Ideally, any progress we also consistently apply to CL Solana.

Here is a nice article on [Better project environment management with direnv and asdf](https://blog.mikecordell.com/2021/12/18/better-project-environment-management-with-direnv-and-asdf/).